cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
 :context: odo-release-notes
 toc::[]
 
-include::modules/openshift-developer-cli-odo-notable-features.adoc[leveloffset=+1]
+[id="odo-notable-improvements_{context}"]
+== Notable improvements in {odo-title}{nbsp}{odo-ver}
 
-include::modules/openshift-developer-cli-odo-getting-support.adoc[leveloffset=+1]
+* IBM Z and PowerPC architecture binaries are now available.
+* `odo catalog` has been improved to provide more useful output information.
+* Information prompts have been added to the `odo service create` command, providing possible next step scenarios.
+* Log verbosity can now be set with an environment variable `ODO_LOG_LEVEL`.
+* Use `odo preference set PushTimeout <seconds>` to specify the number of seconds `odo` waits for a component pod to be deployed before the command fails.
+* Overall documentation improvements. The documentation now includes a description of {odo-title}'s inner architecture.
 
-include::modules/openshift-developer-cli-odo-technology-preview.adoc[leveloffset=+1]
+////
 
-include::modules/openshift-developer-cli-odo-known-issues.adoc[leveloffset=+1]
\ No newline at end of file
+= Notable features in odo for the OCP 4.2 release
```

Comment:
This section is not rendering in the preview build. I think you need to make this an h2 (or possibly add a module).

Reply:
it is not supposed to be rendered. It is commented out (//// text ////). Leaving it here for future reference as per @Preeticp request.